docs/zed/api.md

```diff
@@ -43,13 +40,55 @@ POST /pool
 | Name | Type | In | Description |
 | ---- | ---- | -- | ----------- |
 | name | string | body | **Required.** Name of the pool. Must be unique to lake. |
-| layout.order | string | body | Order of value storage in pool. Possible values: desc, asc. Default: asc. |
-| layout.keys | [string] | body | Primary key(s) of pool. Default: ts. |
+| layout.order | string | body | Order of storage by primary key(s) in pool. Possible values: desc, asc. Default: asc. |
+| layout.keys | [[string]] | body | Primary key(s) of pool. The element of each inner string array should reflect the hierarchical ordering of named fields within indexed records. Default: [[ts]]. |
 | thresh | int | body | The size in bytes of each seek index. |
 
+**Example Request**
+
+```
+curl -X POST \
+     -H 'Content-Type: application/json' \
+     -d '{"name": "inventory", "layout": {"keys": [["product","serial_number"],["warehouse"]]}}' \
```

Comment:
I like where your head is at with this but I do not think we want to highlight primary/secondary keys since I'm not sure this functionality is supported throughout the code base.

Reply:
@mattnibs: Ok, that sounds seems like a valid concern. FWIW, the help text in `zed create -h` speaks of “comma-separated pool keys” (plural) so it’s not a well-hidden secret overall. But if you think showing it in an actual example is taking it too far, I’d be happy to trim the example to show only the primary.

That said, part of my motivation for showing it how I did was to make sense of the array-of-arrays that `layout.keys` happens to be. In addition to it having been described incorrectly in the prior doc revision as only a single-dimensional array, even after I saw via Wireshark sniffing that it was a double array, it didn’t make sense to me until @jameskerr explained how it was used to handle keys in embedded objects. But if you’d rather have it show an array-of-arrays and not have it really be obvious why it’s like that, I can live with that.